tests/electrum.rs

```diff
@@ -137,3 +140,44 @@ fn test_electrum() -> Result<()> {
 
     Ok(())
 }
+
+/// Test the Electrum RPC server using an headless Electrum wallet
```

Comment:
The comment here should be updated, "using a raw TCP socket" or something similar